calnex/export/log.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright (c) Facebook, Inc. and its affiliates.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package export
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+)
+
+var Std = JSONLogger{out: os.Stdout}
```

Comment:
I would move this to be more explicit as a part of Export() argument.

Reply:
Ok, moved.